start/sw/gui-app/electron.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron
+subsection: gui-app
+section: start-sw
+description: 
+---
+
+# Create an application using Electron
+
+[Electron][0] is cross platform framework for creating desktop applications using
+web technologies. It is developed by GitHub and released under the MIT license.
+
+## Installing the development packages
+
+Electron uses web technologies so before installing Electron you will need to
+install NodeJS and NPM. You can install them by running:
+
+```
+$ sudo dnf install nodejs
```

Comment:
I'd rather avoid content  duplication completely. Just linking the page is fine with me.

Reply:
Yes, developers know how to hande that. Simplicity is the key. They can even have already nodejs installed and may just wonder npm packages are needed etc.. No need to have every guide here full-featured. It'd not supposed to be a documentation.